lib/tests/test_local_working_copy.rs

```diff
@@ -2131,3 +2133,175 @@ fn test_snapshot_max_new_file_size() {
         UntrackedReason::FileTooLarge { .. }
     );
 }
+
+#[test]
+fn test_snapshot_symlink_use_forward_slash() {
+    if !file_util::check_symlink_support().unwrap() {
+        eprintln!("Symlink not supported. Skip the test.");
+    }
+    let mut test_workspace = TestWorkspace::init();
+    let workspace_root = test_workspace.workspace.workspace_root().to_owned();
+    let target = repo_path("target/link/target.txt");
+    let target_path = target.to_fs_path(&workspace_root).unwrap();
+    std::fs::create_dir_all(target_path.parent().unwrap()).unwrap();
+    std::fs::write(&target_path, "a\n").unwrap();
+    let link = repo_path("link/link.txt");
+    let link_path = link.to_fs_path(&workspace_root).unwrap();
+    let link_contents = file_util::relative_path(link_path.parent().unwrap(), &target_path);
```

Comment:
nit: use literal string since this test doesn't check if the symlink can be traversed.